src/registry_schemas/example_data/schema_data.py

```diff
@@ -49,7 +49,28 @@
     'lastLedgerTimestamp': '2019-04-15T20:05:49.068272+00:00',
     'lastPreBobFilingTimestamp': '2019-04-15T20:05:49.068272+00:00',
     'legalName': 'legal name - CP1234567',
-    'legalType': 'CP'
+    'legalType': 'CP',
+    'state': 'ACTIVE',
+    'goodStanding': True,
+    'complianceWarnings': [
+        {
+            'code': 'INVALID_LEGAL_STRUCTURE_DIRECTORS',
```

Comment:
Do we have an enum for this, or should we just leave it as a string for now?

Reply:
No We dont have as of now. We will be creating one when we start working on non compliance

Reply:
Sounds good.